### PR TITLE
Increase search items limit from 1k to 10k

### PR DESCRIPTION
**Pull request by zlayne**
/hold

### CODE REVIEW

**Comment by jlpadilla** [comment]
@zlayne Could we use an env variable to change this limit?
We want to keep increasing this limit in the near future as performance allows. An ENV would help us test higher limits or we can advise customers to reduce it if we encounter any problems.

**Comment by zlayne** [comment]
@jlpadilla I have included the changes to use `SEARCH_QUERY_LIMIT` as an env variable to manage the limit.

File: frontend/src/routes/Home/Search/SearchPage.tsx [hunk 1]
```diff
@@ -131,19 +132,20 @@ function RenderSearchBar(props: {
...
-  }, [currentSearch])
+  }, [currentSearch, searchQueryLimit])
 
   const searchCompleteResults = useSearchCompleteQuery({
     skip: !currentSearch.endsWith(':') && !operators.some((operator: string) => currentSearch.endsWith(operator)),
     client: process.env.NODE_ENV === 'test' ? undefined : searchClient,
     variables: {
       property: searchCompleteValue,
       query: searchCompleteQuery,
+      limit: 10000,
```
Comment (jlpadilla) [inline frontend/src/routes/Home/Search/SearchPage.tsx:148]: Should read from env. `useSearchQueryLimit()`

Comment (zlayne) [reply frontend/src/routes/Home/Search/SearchPage.tsx]: updated.

**Review by jlpadilla — APPROVED** [review]
/lgtm

**Pull request merged**